server/common.ts

```diff
@@ -115,6 +116,7 @@ class Config implements IConfig.Main {
 		if (config.maxTeamSize) {
 			this.maxTeamSize = config.maxTeamSize;
 		}
+		this.enableQRCode = !!config.enableQRCode;
```

Comment:
Can you please change this to being a setting that doens't require a redeploy just like the "Teams enabled?" option currently in the admin settings?

Reply:
@petschekr all set, lmk if it looks good